omod/src/main/java/org/openmrs/module/adminui/page/controller/systemadmin/accounts/AccountPageController.java

```diff
@@ -127,17 +127,22 @@ public String post(PageModel model, @MethodParam("getAccount") @BindParams Accou
         Map<String, String[]> parameterMap = request.getParameterMap();
         for(Extension ext : customUserPropertyEditFragments) {
             if (StringUtils.equals(ext.getExtensionParams().get("type").toString(), "userProperty")) {
-                String userPropertyName = ext.getExtensionParams().get("userPropertyName").toString();
-                String[] parameterValues = parameterMap.get(userPropertyName);
-                if (parameterValues != null && parameterValues.length > 0) {
```

Comment:
It is hard to tell what exactly changed when you combine formatting with the actual changes.

Reply:
fixed it please check now